README.md

```diff
@@ -46,6 +46,13 @@ $ botnim --help
 2. In case of changes to the vector stores, remove them in the OpenAI account playground.
 Either:
 3. `botnim sync {staging/production} {budgetkey/takanon}` to sync the specifications with the OpenAI account.
+   - Without flags: updates only the assistant's configuration and instructions
+   - Use `--replace-context` to recreate vector stores from scratch (useful when context files have changed)
+   - Use `--update-common-knowledge` to update only the common knowledge files without modifying the assistant
```

Comment:
I think a better interface would have been something like:
`--update-kb common-knowledge.md`  (kb as in knowledge-base)
This allows for a more flexible command line interface instead of hard coding a filename in your code.

Reply:
We can even extend this to allow for two special values: `none` and `all`

`none` is the default, and it means "don't update knowledge bases if they already exist".
`all` is the opposite (i.e. update all knowledge bases)

This allows for one parameter instead of needing the `--update-instructions` parameter too.

Reply:
And also deprecating `--replace-context` obviously